model/ExpandedLicensing/Vocabularies/NoAssertionLicense.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoAssertionLicense
+
+## Summary
+An Individual Value for License when no assertion can be made about it's actual value.
+
+
+## Description
+
+NoAssertion should be used if the SPDX creator has attempted to but cannot reach a reasonable objective determination;
+the SPDX creator has made no attempt to determine this field; or
+the SPDX creator has intentionally provided no information (no meaning should be implied by doing so).
+
+## Metadata
+
+- name: NoAssertionLicense
+- type: License
```

Comment:
I think this should extend `AnyLicenseInfo` or maybe `ExtendableLicense`. It then would also not have a licenseText, that would need to be overwritten with weird text.

Reply:
Agree - I'll include a fix in the next update